packages/manager/apps/dedicated/client/app/ip/translations/Messages_fr_FR.json

```
@@ -221,11 +221,12 @@
   "ip_description_field": "Description",
   "ip_description_new_title": "Ajouter une description",
   "ip_description_edit_title": "Editer la description",
-  "ip_edit_or_assosiate_title": "Editer/ Associer une organisation",
-  "ip_edit_or_assosiate_editor_tab": "Editer",
-  "ip_edit_or_assosiate_asso_tab": "Associer une organisation",
+  "ip_edit_or_assosiate_description": "Cette section vous permet de personnaliser les informations publiques sur votre bloc IP (informations présentes sur la base whois).",
+  "ip_edit_or_assosiate_title": "Voir/Modifier les informations sur le bloc IP",
+  "ip_edit_or_assosiate_editor_tab": "Réseau",
+  "ip_edit_or_assosiate_asso_tab": "Organisation",
```

Maybe the key should be named differently ?

```suggestion
  "ip_edit_or_assosiate_corporation_tab": "Organisation",
```